projects/elements-demo/e2e/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.base.json",
+  "extends": "../../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../../out-tsc/e2e",
```

Comment:
Shouldn't this NOT exist after we removed protractor?

Reply:
Yeah, removed